scripts/processWork.zs

```diff
@@ -327,6 +327,27 @@ function workEx(machineNameAnyCase as string, exceptionsAnyCase as string,
       // mods.ic2.ThermalCentrifuge.addRecipe([IItemStack[] outputs, IIngredient input, @Optional int minHeat);
       mods.ic2.ThermalCentrifuge.addRecipe(outputItems, inputIngr0);
       return machineName;
+    } //Mass Spectrometer dirty gem no liquid
```

Comment:
Conventionally, end of block with a comment like `} // comm` addressing comment to previous block, not next one.
Move code and add space after `//`:
```ts
}

// Mass Spectrometer dirty gem no liquid`
```